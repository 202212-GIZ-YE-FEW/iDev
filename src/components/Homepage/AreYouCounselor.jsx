import { withTranslation } from "next-i18next";

import Button from "@/components/ui/Button";
function AreYouCounselor({ t }) {
    return (
        <>
            <div className='bg-light-white flex flex-col items-center justify-center text-center py-10 px-5 rounded-md space-y-6 drop-shadow-lg'>
                <span className='uppercase text-4xl'>
                    {t("askIfCounselor")}
                </span>
                <span className='text-base rtl:text-xl text-black'>
                    {t("askCounselor")}
                </span>
                <div className=''>
                    <Button
                        content={t("learnMore")}
                        textTransform='uppercase'
                        filled='true'
                        size='large'
                        fontSize='text-lg md:text-xl lg:text-2xl'
                        radius='md'
                    />
                </div>
            </div>
        </>
    );
}
export default withTranslation("home")(AreYouCounselor);
