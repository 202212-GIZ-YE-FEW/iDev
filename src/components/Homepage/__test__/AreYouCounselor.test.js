import renderer from "react-test-renderer";

import AreYouCounselor from "../AreYouCounselor";

it("renders correctly", () => {
    const tree = renderer.create(<AreYouCounselor />).toJSON();
    expect(tree).toMatchSnapshot();
});
