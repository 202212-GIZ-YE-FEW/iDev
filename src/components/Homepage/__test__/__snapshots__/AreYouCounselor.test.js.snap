// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="bg-light-white flex flex-col items-center justify-center text-center py-10 px-5 rounded-md space-y-6 drop-shadow-lg"
>
  <span
    className="uppercase text-4xl"
  >
    askIfCounselor
  </span>
  <span
    className="text-base rtl:text-xl text-black"
  >
    askCounselor
  </span>
  <div>
    <a
      href="/requirements"
      onClick={[Function]}
      onMouseEnter={[Function]}
      onTouchStart={[Function]}
    >
      <div
        className="relative self-center"
      >
        <button
          className="transform hover:bg-yellow transition duration-500  hover:animate-bounce hover:scale-75 active:bg-cyan focus:outline-none focus:ring focus:ring-cyan
                font-normal whitespace-nowrap cursor-pointer  uppercase text-lg md:text-xl lg:text-2xl rounded-md px-[25px] md:px-[35px] py-[8px] font-medium bg-cyan text-black "
          disabled={false}
          onClick={[Function]}
        >
          learnMore
        </button>
      </div>
    </a>
  </div>
</div>
`;
